### PR TITLE
feat: separate the isolation and forensic policy into tier0 and tier1

### PR DESCRIPTION
**Pull request by wu-cl**

### CODE REVIEW

File: plugin/tower/pkg/controller/policy/controller.go [hunk 1]
```diff
@@ -46,6 +46,8 @@ import (
 const (
 	SecurityPolicyPrefix             = "tower.sp-"
 	IsolationPolicyPrefix            = "tower.ip-"
+	IsolationPolicyIngressPrefix     = IsolationPolicyPrefix + "ingress-"
```
Comment (zwtop) [inline plugin/tower/pkg/controller/policy/controller.go:49]: nit: use "." as separator in prefix

Comment (wu-cl) [reply plugin/tower/pkg/controller/policy/controller.go]: done

**Review by KeyongSun — APPROVED** [review]
@echkenluo 看一下 datapath 是否有什么影响的地方

File: plugin/tower/pkg/controller/policy/controller.go (hunk 1 above)
Comment (zwtop) [reply plugin/tower/pkg/controller/policy/controller.go]: index function needs to update
https://github.com/everoute/everoute/pull/232/files#diff-9c9bbeae13e01d343396480c47dc33d4841d8845e278880bbd906ac58752103dR281-R288

Comment (wu-cl) [reply plugin/tower/pkg/controller/policy/controller.go]: done

**Pull request merged**